pkg/server/sandbox_run.go

```diff
@@ -137,7 +137,7 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		// In this case however caching the IP will add a subtle performance enhancement by avoiding
 		// calls to network namespace of the pod to query the IP of the veth interface on every
 		// SandboxStatus request.
-		sandbox.IP, sandbox.CNIResult, err = c.setupPod(ctx, id, sandbox.NetNSPath, config)
+		sandbox.IP, sandbox.AdditionalIPs, sandbox.CNIResult, err = c.setupPod(ctx, id, sandbox.NetNSPath, config)
```

Comment:
starting to look a bit odd here might want to just pass a sandbox to setupPod.. and rename it to setupPodSandbox

Reply:
It only setup pod network.

Changed to `setupPodNetwork` and `teardownPodNetwork`.

Reply:
by odd I also meant passing sandbox.NetNSPath... then lval assigning 3 sandbox values on the return, not just the name.

Reply:
Will do.